curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-bracket-notation-to-find-the-first-character-in-a-string.english.md

```diff
@@ -39,18 +39,12 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var firstLetterOfFirstName = "";
-var firstName = "Ada";
-
-firstLetterOfFirstName = firstName[0];
```

Comment:
The hint on line 19 no longer makes sense if we remove this. Either this could be moved into the description or the hint needs changing.

Reply:
Good catch. I will try to move the example into the description section.

Reply:
I added an example based on the challenge description.